tests/test_ec2_manager.py

```diff
@@ -16,6 +16,22 @@ def test_list_security_groups(em):
     assert em.list_security_groups()
 
 
+def test_list_instances_with_status(em):
+    pprint(em.list_instances_with_status())
+
+
+def test_delete_unused_key_pairs(em):
+    em.delete_unused_key_pairs()
```

Comment:
I think there should be a statement to check if unused key pairs are properly deleted

Reply:
KMS related unit tests need more assertions and 'Given When Then' formatting. Your comment will be reflected soon